### ISSUE TITLE
Tweet is not correct

### ISSUE THREAD

**Issue by christophrumpel** [christophrumpel/larastreamers#190]
Today a tweet about an upcoming stream was published. The stream was already changed on Youtube, but still wrong data on Larastreamers.

Let's update a stream before we send a tweet.

**Comment by mathiasonea** [comment]
Would we do that by executing the `ImportVideoAction` Job with the existing stream to make sure the stream is up to date?

maybe something like this in the `handle()` method of the `ImportVideoAction`
```php
    $this->makeSureStreamIsUpToDate();
```

and its implementation down below

```php
    protected function makeSureStreamIsUpToDate(): void
    {
        $this->stream = app(ImportVideoAction::class)->handle(
            youTubeId: $this->stream->youtube_id,
            languageCode: $this->stream->language_code,
            approved: $this->stream->isApproved(),
            submittedByEmail: $this->stream->submitted_by_email,
        );
    }
```

Or do you think it might make sense to introduce another Job `UpdateVideoAction` which receives only the `youtube_id` to make sure we don't have to think about the other parameters when we only want to make sure that specific Video's data is getting updated in our database?

**Comment by mathiasonea** [comment]
@christophrumpel  This new Action should be called in the handle() methods in the `TweetStreamIsLiveJob` and `TweetStreamIsUpcomingJob`.

**Comment by christophrumpel** [comment]
Hey, thank YOU for tackling this in the first place :-) Yeah, let's re-use this action for now with the tweet jobs. Once it is done, we still can think of a "cleaner" solution.

Let me know if you need help with the implementation or the tests. thx Mathias

**Comment by christophrumpel** [comment]
Hey @mathiasonea, did you start working on this? No issue if not, please just let me know.

**Comment by mathiasonea** [comment]
@christophrumpel no, unfortunately I didn't have as much time for this as I'd wanted to..

**Comment by christophrumpel** [comment]
No worries, just good to know 👍
